src/components/search/search.test.js

```diff
@@ -404,4 +392,154 @@ context("Test for Search component", () => {
       }
     );
   });
+  describe("Accessibility tests for Search", () => {
+    it.each(testData)(
+      "should check accessibility for Search with placeholder using %s as special characters",
+      (placeholder) => {
+        CypressMountWithProviders(
+          <SearchComponent placeholder={placeholder} />
+        );
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it("should check accessibility for Search with defaultValue prop", () => {
+      CypressMountWithProviders(<Search defaultValue={testCypress} />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should check accessibility for Search with value prop", () => {
+      CypressMountWithProviders(<SearchComponent value={testCypress} />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should check accessibility for Search with id prop", () => {
+      CypressMountWithProviders(<SearchComponent id={testCypress} />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should check accessibility for Search with name prop", () => {
+      CypressMountWithProviders(<SearchComponent name={testCypress} />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should check accessibility for Search with aria-label prop", () => {
+      CypressMountWithProviders(<SearchComponent aria-label={testCypress} />);
+
+      cy.checkAccessibility();
+    });
+
+    it("should check accessibility for searchButton", () => {
+      CypressMountWithProviders(<SearchComponent searchButton />);
+
+      cy.checkAccessibility();
+    });
+
+    it.each(["34%", "70%"])(
+      "should check accessibility for Search with searchWidth prop set to %s",
+      (widthInPercentage) => {
+        CypressMountWithProviders(
+          <SearchComponent searchWidth={widthInPercentage} />
+        );
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each(["475px", "250px"])(
+      "should check accessibility for Search with searchWidth prop set to %s",
+      (width) => {
+        CypressMountWithProviders(<SearchComponent searchWidth={width} />);
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it.each(["10%", "34%", "70%", "100%"])(
+      "should check accessibility for Search with maxWidth prop set to %s",
+      (widthInPercentage) => {
+        CypressMountWithProviders(
+          <SearchComponent maxWidth={widthInPercentage} />
+        );
+
+        cy.checkAccessibility();
+      }
+    );
+
+    it("should check accessibility for Search with tabIndex prop", () => {
+      CypressMountWithProviders(<SearchComponent tabIndex={-5} />);
+
+      cy.checkAccessibility();
+    });
+
+    // FE-4670
```

Comment:
Is this comment meant to be here?

Reply:
Thanks for reviewing it Dan, this comment is for the purpose of skipping the tests below as there are still accessibility violations in search. And this is the ticket in our backlog.

Reply:
Didn't see the `.skip` 😆